src/test.js

```diff
@@ -877,3 +885,10 @@ function notifyTestsRan( module, skipped ) {
 		}
 	}
 }
+
+function notifyTestsNotRan( module ) {
```

Comment:
Should this function have a more accurate name? It's not really notifying anything. It's accumulating the total count of tests not run, isn't it?

Reply:
Changed to `updateTestsNotRan`